src/test/java/org/dita/dost/ITContentUplevels.java

```diff
@@ -0,0 +1,114 @@
+package org.dita.dost;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.dita.dost.AbstractIntegrationTest.Transtype.XHTML;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+public interface ITContentUplevels {
+  AbstractIntegrationTest builder();
+
+  @Test
+  default void uplevels1() {
+    var outerFiles = getListOfPotentialOuterFilesInTmpAndOut();
+    cleanUpAnyLeftovers(outerFiles);
+
+    assertThatThrownBy(() ->
```

Comment:
My preference would be to not add additional dependencies if possible, even if they are for testing. Could this be replaced by

```java
var e = assertThrows(AssertionError.class, () -> ...);
assertTrue(e.getMessage().contains("Missing file"));
```

I admin it's not as fluent as the AssertJ, but